pkgs/tools/networking/phantomsocks/default.nix

```
@@ -16,6 +24,17 @@ buildGoModule rec {
 
   vendorHash = "sha256-c0NQfZuMMWz1ASwFBcpMNjxZwXLo++gMYBiNgvT8ZLQ=";
 
+  ldflags = [
+    "-s" "-w"
+  ];
+  # We need libpcap to build pcap version. #TODO Not found pcap.h when use nativeBuildInputs, but AUR can use in makedepends.
+  buildInputs = [ ]
+    ++ lib.optionals pcap [ libpcap ];
+  tags = [ ]
+    ++ lib.optionals pcap [ "pcap" ]
+    ++ lib.optionals rawsocket [ "rawsocket" ]
+    ++ lib.optionals windivert [ "windivert" ];
```

```suggestion
  buildInputs = lib.optional withCcap libpcap;
  tags = lib.optional withPcap "pcap"
    ++ lib.optional withRawsocket "rawsocket";
```
libraries go to buildInputs, thats not a todo

---

Does recommend to use "withPcap" this format?